### PR TITLE
Remove UGH

### PR DESCRIPTION
**Pull request by matthias-ronge**

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/helper/metadata/LegacyDocStructHelperInterface.java [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.helper.metadata;
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/java/org/kitodo/helper/metadata/LegacyDocStructHelperInterface.java]: Shouldn't this go to the legacy-package?

File: Kitodo/pom.xml [hunk 2]
```diff
@@ -767,15 +767,6 @@
                 <artifactId>maven-war-plugin</artifactId>
                 <version>3.2.2</version>
                 <configuration>
-                    <webResources>
-                        <resource>
-                            <directory>${parent.basedir}/OpacPica-Plugin/target</directory>
```
Comment (Kathrin-Huber) [inline Kitodo/pom.xml]: Can this already be removed? Even when, I think this should be done in another pull Request.

**Comment by Kathrin-Huber** [comment]
Please resolve conflicts

File: Kitodo/src/main/java/org/kitodo/helper/metadata/LegacyDocStructHelperInterface.java (hunk 1 above)
Comment (matthias-ronge) [reply Kitodo/src/main/java/org/kitodo/helper/metadata/LegacyDocStructHelperInterface.java]: Good catch! 👍

File: Kitodo/pom.xml (hunk 2 above)
Comment (matthias-ronge) [reply Kitodo/pom.xml]: It no longer works without dependency on the UGH.

Doing thi in another pull request is going to be difficult, I'm afraid. I mean, the newly implemented validation (Pull Request #2151) is based on this branch. How can that still be merged then?

What exactly should I do?

**Comment by matthias-ronge** [comment]
The pull request is again put in pieces.
